js/src/search-control.js

```diff
@@ -0,0 +1,103 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import GridIconSearch from 'gridicons/dist/search';
+import PropTypes from 'prop-types';
+
+/**
+ * WooCommerce dependencies
+ */
+import { TextControl } from '@woocommerce/components';
+
+/**
+ * Action Scheduler admin screen component
+ */
+class SearchControl extends Component {
```

Comment:
Its a shame the `@woocommerce/components` search isn't amenable to just searching with simple strings.

You can have a play at http://localhost:6007/?path=/docs/woocommerce-admin-components-search--basic after running `npm run storybook`.

Maybe its possible to run a simplified version of it or perhaps `SelectControl`. I'm saying this because those elements are in use on the Customers page and visual alignment in design would be nice.

![image](https://user-images.githubusercontent.com/1922453/137227424-ab6d31b0-387f-4f6a-a0ac-172327cd642a.png)

Did you give it a try? At the very least maybe there are some classnames you can apply to get some style consistency

Reply:
> Did you give it a try? At the very least maybe there are some classnames you can apply to get some style consistency

I did. I think both require an api endpoint to poll for auto suggest. Due to the duplication in action hooks and args, auto suggesting individual actions isn't going to be helpful to the user in most cases. I also looked at `@wordpress/components` `SearchControl` and it wasn't suitable either.


I'll double check the WCA controls tomorrow and have a look at the class names.

Reply:
@psealock I had to go with raw HTML elements to achieve the consistent appearance due to the Muriel classes in the `<TextControl />`. While it would be preferable to not use those elements, I think the consistent appearance is worth it in this case. Thanks for that suggestion.